### PR TITLE
add: [blogpost] Polkadot Generative Drops 

### LINKED ISSUE #8753: [blogpost] Polkadot Generative Drops
date
---
18/12/2023

title
---
Polkadot Generative Drops on KodaDot
1_z3_cTMw_epywEjVozNY1Ww (1)

The past few weeks have been quite eventful at KodaDot as they have had four significant drops of generative art. This innovative concept on Polkadot has garnered a lot of attention. Let's take a quick look at these fantastic drops and get a glimpse of what's coming up next!

What is generative art?
---
Generative art is a unique form of digital art created using algorithms and code. It's a combination of technology and creativity, resulting in artworks that are both unique and intriguing. Instead of NFTs, one can "mint the code" in generative art. Those interested in the NFT from the collection will generate their own unique variation that will be minted afterward.

At KodaDot, we're giving away generative art for free as NFTs. You can make your own art before minting it, which is really fun. You can try different styles and pick the one you love the most. It's like being an artist and a collector at the same time!

Newest Drops
---
"Every individual is entitled to grab a single NFT through their wallet. The most exciting part is that you will be a part of creating it. It is fascinating to witness the various designs people come up with. This week, we had two remarkable drops, each with unique features and limited availability. It's an ideal opportunity for devoted collectors!"

### 1) Wall Street Art

Artist: [Teraki](https://twitter.com/terakiart)
1_iO9RGt0lb_ZCW7bt3ZWLEg (1)

Inspiration from the grit and vibrancy of the city streets, where walls become dynamic canvases. The generative algorithm driving this collection incorporates the fluidity of spray paint, the spontaneity of street tags, and the bold strokes of urban murals. Every artwork in this series is a true original. The algorithm injects an element of randomness and unpredictability, ensuring that no two pieces are alike.

Check the collection [HERE](https://kodadot.xyz/ahp/collection/49). The collection was minted out in 35 minutes.

### 2) .motherboard
Artist: [Conatus](https://twitter.com/dotconatus)

1_BoTO7UhR2VDVeHfyo1gqmw (1)

.motherboard reimagines the motherboard as refined art, expressing it as dynamic digital art. The sophisticated arrangement of the motherboard’s components and circuit patterns, striving for efficiency, can be considered a marvelous fusion of technology and art.

Check the collection [HERE](https://kodadot.xyz/ahp/collection/50). The collection was minted out in less than an hour.

Previous Polkadot Drops
In the last few weeks, we’ve done another three of these free drops, and let me tell you, it’s been a blast! People are loving it. And guess what? We’re not stopping. We’re planning to do even more drops and give people a proper collecting experience.

### 3) Swirls
Artist: [Sabha](https://twitter.com/sabhab1)

1_kQFJoZ4EkDfOvVZQ1xgQbQ (1)

Check the collection [HERE](https://kodadot.xyz/ahp/collection/40).

Swirls are mesmerizing art pieces. Each of them is full of dots *I guess we all love Polkadot( Each piece in the collection is born from a generative algorithm that mirrors the spiraling sequences found in galaxies and growth rings, revealing the underlying order in apparent randomness. The collection’s palette is a tribute to these organic forms, with colors ranging from earthy and subtle to vibrant and dynamic, reflecting the diverse expressions of natural swirls.

In this artistic endeavor, the algorithm acts as a bridge between the digital realm and the recursive beauty of the natural world. Collectors are invited to own a piece of this perpetual motion, where each NFT is a snapshot of creation, a harmonious blend of art and mathematics that resonates with the rhythm of life’s designs.

### 4) Snowflakes
Artist: [Aurora](https://twitter.com/poppyseedDev)
1_UPnBpSEGc5fmP2gr7LrecQ (1)

Every snowflake must go through “Fractal Quest,” which consists of five quests that will create a whole story about these NFTs.

Check the collection [HERE](https://kodadot.xyz/ahp/collection/46).

### 5) Pare1d0scope
Artist: [sha22.art](http://sha22.art/)

1_JlDFIcQ9IYUTLf7gWisxaQ (1)

Check the collection [HERE](https://kodadot.xyz/ahp/collection/38).

This project is a collaboration between [sha22.art](http://sha22.art/), Antonino, and KodaDot. Anchored in a not-so-binary world, the pare1d0scope generates unique patterns of 4-bit squares by applying a generative mix of symmetry, ordering, subdividing, and diagonal adjustments. This collection features a special color palette to celebrate Generative Art’s landing in the Polkadot space! Play with the NFTs even after minting; each NFT is interactive!

What's next?
---
Collect the NFTs because we are preparing for you a collecting experience. Some of the drops will be rarer than others, and only those holders who own NFT from previous collections will be able to mint it.

Why should I list my NFT for sale?
---
- Profit Potential: The primary reason for listing NFTs soon after purchase is the potential for profit. If the NFT has gained popularity or increased in value since its acquisition, selling it quickly can yield a significant return on investment.
- Market Trends and Hype: The value of NFTs can be highly dependent on current trends and hype. Listing them while they are still in high demand ensures that the seller capitalizes on these market sentiments.
- Volume growth: Listing an NFT soon after purchase can contribute to the overall activity and health of the NFT collection. And with that, we all benefit.
- Experimentation and Learning: For newcomers to the NFT space, quick buying and selling can serve as a learning experience, helping them understand market dynamics without committing to long-term investments.
- Traders will be rewarded too: We encourage you to trade your newly obtained NFTs. List it for sale, or buy a new one, and contribute to the growing volume of the collection. Remember, traders will be rewarded on KodaDot.

It’s an amazing time at KodaDot, mixing art, code, and fun. I can’t wait to see what awesome art everyone’s going to create and collect next!

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

new url **/blog/polkadot-generative-drops-on-kodadot**

- [x] Closes #8753

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-12-27 at 11 32 55

## Copilot Summary

copilot:summary

copilot:poem

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Review by roiLeo — APPROVED** [review]

File: content/blog/polkadot-generative-drops-on-kodadot.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+This project is a collaboration between [sha22.art](http://sha22.art/), Antonino, and KodaDot. Anchored in a not-so-binary world, the pare1d0scope generates unique patterns of 4-bit squares by applying a generative mix of symmetry, ordering, subdividing, and diagonal adjustments. This collection features a special color palette to celebrate Generative Art’s landing in the Polkadot space! Play with the NFTs even after minting; each NFT is interactive!
+
+## What's next?
+
+Collect the NFTs because we are preparing for you a collecting experience. Some of the drops will be rarer than others, and only those holders who own NFT from previous collections will be able to mint it.
+
+## Why should I list my NFT for sale?
+
+- Profit Potential: The primary reason for listing NFTs soon after purchase is the potential for profit. If the NFT has gained popularity or increased in value since its acquisition, selling it quickly can yield a significant return on investment.
```
Comment (JustLuuuu) [inline content/blog/polkadot-generative-drops-on-kodadot.md:79]: maybe remove all "-" in this paragraph. It doesn't look good published. Maybe do it in bold:

**Market Trends and Hype**: The value of NFTs can be highly dependent on current trends and hype. Listing them while they are still in high demand ensures that the seller capitalizes on these market sentiments.
**Volume growth**: Listing an NFT soon after purchase can contribute to the overall activity and health of the NFT collection. And with that, we all benefit.
**Experimentation and Learning**: For newcomers to the NFT space, quick buying and selling can serve as a learning experience, helping them understand market dynamics without committing to long-term investments.
**Traders will be rewarded too:** We encourage you to trade your newly obtained NFTs. List it for sale, or buy a new one, and contribute to the growing volume of the collection. Remember, traders will be rewarded on KodaDot.

**Review by JustLuuuu — CHANGES_REQUESTED** [review]
Just a small cosmetic change pls.

**Comment by yangwao** [comment]
we can do follow-ups with @JustLuuuu comments 
merging bc of release 

Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.66 USD/DOT ~ 3.464 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xa356667707f098f8ee525bc365c8ab5f6cddaf6221f15efca9f0b88d034938bc](https://polkadot.subscan.io/extrinsic/0xa356667707f098f8ee525bc365c8ab5f6cddaf6221f15efca9f0b88d034938bc)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**